src/components/routerView.browser.spec.ts

```diff
@@ -115,14 +116,11 @@ test('updates components when route changes', async () => {
   expect(app.html()).toBe('Foo')
 })
 
-test.each([
-  defineAsyncComponent(() => import('./helloWorld')),
-  () => import('./helloWorld'),
```

Comment:
This is an important thing to note. Previously we supported any async import as a component by automatically wrapping any function in `defineAsyncComponent`. 

I'm removing that in this PR. Now if you want to use an async component you should declare it as such by using vue's `defineAsyncComponent` utility. 

This will give us more control over components without transforming them (sometimes unnecessarily). Since we cannot detect if a component function is an async import without first calling it (which makes the import call).